packages/components/fields/async-creatable-select-field/src/async-creatable-select-field.tsx

```diff
@@ -0,0 +1,480 @@
+import {
+  Component,
+  isValidElement,
+  type ReactElement,
+  type ReactNode,
+} from 'react';
+import { type GroupBase, type ActionMeta } from 'react-select';
```

Comment:
Maybe we could standardise and use the `type` directive outside the brackets if all what we're importing are types (just like in the next line).
This way, we would only need to repeat in in case we're importing several types from a package where we also import components (like in the previous line).

What do you think?

Reply:
I completely agree